src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-service-broker-integration.mdx

```diff
@@ -0,0 +1,423 @@
+
+    awk ‘{print $2}’ | sed ‘s/[",]//g’) | egrep “"name":|"unique_id":” |
+
+    sed ‘s/[",]//g’ | tr -s " " | awk ’ {name=$0; getline; printf(\t%-40s %-40s \n”,name,$0)}’
+   ```
+
+There are two new properties labeled pre-1.12.12 plan? and Plan Guid Override (broker 1.12.12 or older) in the plan collection for each plan in the tile configuration. Do not change either of these properties because they are set internally where required.
+
+* In New Relic Service Broker v1.12.13 and later, leave pre-1.12.12 plan unchecked, and Plan Guid Override blank.
+* The tile is supported on Ops Manager v2.9.x, v2.10.x, and v3.0.x.
+* You can install the tile on any of these versions, and upgrade from v1.9.x to any Ops Manager version up to and including v3.x.
+* No upgrade paths are required for older versions of the tile, since versions older than v1.9.0 are not supported.
+* v1.12.6 and later of the tile support migration from older versions of the tile, and preserve existing services and service plans.
+* If you are using tiles older than v1.11.4, you must first upgrade to v1.11.4, then to v1.12.9, then to the latest version of the tile.
+</Collapser>
+
+
+The following table provides version and version-support information about the service broker for VMware Tanzu.
+
+<table>
+    <thead>
+    <tr>
+        <th>
+            Element
+        </th>
+
+        <th>
+            Details
+        </th>
+    </tr>
+    </thead>
+
+    <tbody>
+    <tr>
+        <td>
+            Version
+        </td>
+
+        <td>
+            1.12.33
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Release date
+        </td>
+        <td>
+            August 15, 2023
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Software component version
+        </td>
+        <td>
+            New Relic Service Broker 1.12.33
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Compatible Ops Manager version(s)
+        </td>
+        <td>
+            2.9.x, 2.10.x, and 3.x
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Compatible VMware Tanzu Application Service for VMs versions
+        </td>
+        <td>
+            2.10.x, 2.11.x, 2.12.x, 2.13.x, 3.x, and 4.x
+        </td>
+    </tr>
+    <tr>
+        <td>
+            BOSH stemcell version
+        </td>
+        <td>
+            Ubuntu Jammy
+        </td>
+    </tr>
+    <tr>
+        <td>
+            IaaS support
+        </td>
+
+        <td>
+            AWS, GCP, Azure, and vSphere
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+The current tile removes the `all_open` security group from the tile default security settings. If you are using a previous versions of the tile, make your VMware Tanzu environment more secure by removing the `all_open` security group from the Application Security Group (ASG) settings. The new version of the tile does not open the security, nor does it close the security if it was already open.
```

Comment:
This was initially a call out, but I broke it out. Since it's security recommendation, we could make this a prereq, or add it as a configuration step later in the doc